oidc-wp/openid-connect-generic
# Local Dev/Composer Scripts/Transient Redirection Fixes

### All Submissions:






### Changes proposed in this Pull Request:



Closes # 293.
- Fixes Local Docker wp-env Environment Setup w/ Cleanup
- Fixes Composer Scripts & Redirect After Authentication

### How to test the changes in this Pull Request:

1. Configure plugin to Redirect User Back.
2. Create a Login button on a page.
3. Try to login from the new page.
4. After IDP authentication you should be sent back to the original page.

1. Configure plugin not to Redirect User Back.
2. Login from the page with the Login button from previous test.
3. After IDP authentication you should be sent back to the home page.

### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
- Fix: @timnolte - Fixed invalid State object access for redirection handling.
- Improvement: @timnolte - Fixed local wp-env Docker development environment.
- Improvement: @timnolte - Fixed Composer scripts for linting and static analysis.
